MSAL/src/native_auth/public/MSALNativeAuthChannelType.swift

```diff
@@ -24,8 +24,13 @@
 
 import Foundation
 
+/// The possible Channel Types via which a Code was sent
 @objc
 public enum MSALNativeAuthChannelType: Int {
+    
+    /// Specifies if the channel type is Email
```

Comment:
I don't see any value in this type of documentation/comment. It is really just stating the obvious isn't it?

Reply:
Indeed it's obvious and there has to be something there otherwise it says undocumented. I can add anything but it has to be something

Reply:
Thanks for the explanation. That's annoying but I understand.